### PR TITLE
Type definition for react-responsive

### PR DESCRIPTION
**Pull request by asvetliakov**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

File: react-responsive/react-responsive.d.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// Type definitions for react-responsive 1.1.3
+// Project: https://github.com/contra/react-responsive
+// Definitions by: Alexey Svetliakov <https://github.com/asvetliakov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../react/react.d.ts" />
+
+declare module "react-responsive" {
+    import * as React from 'react';
```
Comment (zhengbli) [inline react-responsive/react-responsive.d.ts]: Use `'` or `"` consistently

File: react-responsive/react-responsive.d.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    namespace MediaQuery {
+        export interface MediaQueryProps {
+            query?: string;
+            // matchers
+            orientation?: "portrait" | "landscape";
+            scan?: "progressive" | "interlace";
+            aspectRatio?: string;
```
Comment (zhengbli) [inline react-responsive/react-responsive.d.ts]: `aspectRatio` is a `string`? The [css spec](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#aspect-ratio) says it consists of two numbers

Comment (asvetliakov) [inline react-responsive/react-responsive.d.ts]: The propTypes in original library define it as string
https://github.com/contra/react-responsive/blob/master/src/mediaQuery.js#L21

**Comment by zhengbli** [comment]
:+1:

**Pull request merged**